ndk/src/relay/pool/index.ts

```diff
@@ -294,7 +294,8 @@ export class NDKPool extends EventEmitter<{
 
         const relaysToConnect = new Set(this.autoConnectRelays.keys());
         this.ndk.explicitRelayUrls?.forEach((url) => {
-            relaysToConnect.add(url);
+            const normalizedUrl = normalizeRelayUrl(url);
```

Comment:
I thought I had added a test around relay URL normalization (maybe as part of the connectivity stuff)? Just saw this in passing and wanted to double check.